src/rpdk/core/contract/resource_client.py

```diff
@@ -419,11 +425,14 @@ def make_request(
         account,
         action,
         creds,
+        type_name,
```

Comment:
the change makes sense, I am a bit unsure why do you need type_name? could you provide more details?

Reply:
Type name will be used for constructing cloudwatch metric name if there are valid logging credential passed in. Without this, wrapper side will fail for NPE when doing so.